app/scripts/models/account.js

```diff
@@ -605,9 +605,9 @@ define(function (require, exports, module) {
      * Check whether all the passed in permissions have been
      * seen previously.
      *
-     * @param {string} clientId
-     * @param {strings[]} permissions
-     * @returns {boolean} `true` if client has seen all the permissions,
+     * @param {String} clientId
+     * @param {Strings[]} permissions
```

Comment:
I realise you're just capitalising what already existed here, but this notation feels a bit funny to me. Just wondering whether something like `@param {Array[String]}` would be less prone to mis-reading at a glance. What do you reckon?